cluster-autoscaler/utils/errors/errors.go

```diff
@@ -131,3 +133,63 @@ func (e autoscalerErrorImpl) Type() AutoscalerErrorType {
 func (e autoscalerErrorImpl) AddPrefix(msg string, args ...interface{}) AutoscalerError {
 	return autoscalerErrorImpl{errorType: e.errorType, wrappedErr: e, msg: fmt.Sprintf(msg, args...)}
 }
+
+// CombineConcurrentScaleUpErrors returns combined scale-up error to report after multiple concurrent scale-ups might haver failed.
+func CombineConcurrentScaleUpErrors(errs []AutoscalerError) AutoscalerError {
```

Comment:
I don't think there's anything scale up specific here, can you rename this to just `CombineConcurrentErrors`? Or even `CombineErrors` if you also remove the word "concurrent" from line 173?

Reply:
Done

Reply:
Actually, since this is in the errors package now, I'd consider making it an even more concise `errors.Combine`. Up to you though.